mapper/orm-coordination-outbox-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/event/impl/OutboxPollingOutboxEventAdditionalJaxbMappingProducer.java

```diff
@@ -64,6 +64,11 @@
 			"        <property name=\"payload\" type=\"materialized_blob\" />\n" +
 			"        <property name=\"retries\" type=\"integer\" />\n" +
 			"        <property name=\"processAfter\" type=\"Instant\" index=\"processAfter\" />\n" +
+			"        <property name=\"status\">\n" +
+			" 			 <type name=\"org.hibernate.type.EnumType\">\n" +
+			" 				 <param name=\"enumClass\">" + OutboxEvent.Status.class.getName() + "</param>\n" +
+			" 			 </type>\n" +
+			"		 </property>\n" +
```

Comment:
Or more simply, and making the property non-nullable:

```suggestion
			"        <property name=\"status\" nullable="false" />\n" +
```